Lab2.ipynb

```diff
@@ -0,0 +1,677 @@
+      "text/plain": [
+       "<Figure size 432x288 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "x = np.linspace(0, 10, 1000)\n",
+    "dist = stats.rayleigh.pdf(x)\n",
+    "\n",
+    "conv1 = np.convolve(dist, dist)\n",
```

Comment:
In 2A you need to convert your pdf to a pmf before convolving, then you can convert back into pdfs at the end. To go from pdf to pmf multiply by your bin size, and to go the other way divide by your bin size. This should fix the normalization of your distributions.